yaksh/templates/yaksh/show_video.html

```diff
@@ -0,0 +1,77 @@
+{% extends "user.html" %}
+{% load custom_filters %}
+
+{% block title %} {{ learning_module.name|title }} {% endblock %}
+
+{% block pagetitle %} {{ learning_module.name|title }} {% endblock %}
+
+{% block usersidebar %}
+<br><br><br>
+<center><h3>Lessons/Quizzes</h3></center>
+<ul class="list">
+{% for unit in learning_units %}
+<span>
+<li>
+{% get_unit_status course learning_module unit user as status %}
+
+{% if unit.id == current_unit.id %}
+    <span class="glyphicon glyphicon-edit" data-toggle="tooltip" title="Currently on">
```

Comment:
This should be  glyphicon glyphicon-pencil. I suggest it would be more appropriate